helper/src/main/java/com/dailylife/helper/tasks/Task.java

```diff
@@ -27,9 +26,7 @@ public void setId(String id) {
         this.id = id;
     }
 
-    public boolean isDone() {
-        return isDone;
-    }
+    public boolean getDone() {return isDone;}
```

Comment:
why changing to getDone() ? what's wrong with isDone() ?

Reply:
BTW, PR name suggests that you are adding UI, that suggests me that there should be no changes in the backend code